lib/plugins/newrelicTransactionManagerPlugin.ts

```diff
@@ -37,7 +38,7 @@ export class NewRelicTransactionManager {
     }
 
     this.isEnabled = isNewRelicEnabled
-    this.transactionMap = new Map()
+    this.transactionMap = new FifoMap(2000)
```

Comment:
there was a potential memory leak in the original implementation, if for whatever reason we do not close the transaction, it will keep hanging in the map forever.